pegasus/sites.v3/code.org/public/js/page/educate_csd.js

```diff
@@ -1,83 +1,86 @@
 const windowWidth = window.innerWidth;
 
 function reorganizeAndWrapSlides(carouselId, cardsList) {
-    const carouselContainer = document.getElementById(carouselId);
-    if (!carouselContainer) {
-        console.error(`Carousel container with id "${carouselId}" not found.`);
-        return;
-    }
+  const carouselContainer = document.getElementById(carouselId);
```

Comment:
I had to fix these linting errors when I merged staging in because the pre-commit hooks had been skipped at some point.